app/assets/stylesheets/govuk_publishing_components/components/_layout-super-navigation.scss

```diff
@@ -85,3 +85,25 @@
     width: 20px;
   }
 }
+
+.gem-c-super-navigation-header__menu-button {
+  font-size: 19px;
+  font-weight: bold;
+  line-height: 1.3;
```

Comment:
I suggest we use `govuk-frontend` mixins for defining font properties `@include govuk-font($size: 19, $weight: bold, $line-height: 1.3);`

Reply:
Good idea - done!